### DISCUSSION TITLE
Suggestion: Uniform sorting rules for the ‘Add to list‘ drop-down list; Provide batch add project to list

### DISCUSSION THREAD

**Discussion by emoker** [community/community#126683] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

As Topic，Suggest that the ‘Add to list’ drop-down list has a uniform sorting rule each time it is opened. Currently the contents are in a different order each time you open it.
![image](https://github.com/community/community/assets/28655577/b9817ee7-b944-4f00-917d-d107af7782d1)

![image](https://github.com/community/community/assets/28655577/1edd3fa7-b71b-468f-b401-8109b60e086b)

Also, would like to provide the ability to batch add projects to the list.

**Comment by Nurtrisariyanto** [comment]
5ezeyzytut8u9u0h hovyovou0u080h swwr5e6z74d58y9vve6zt7vyu0b08
